src/components/NavBar/NavBar.scss

```diff
@@ -1,5 +1,159 @@
 @import url('https://fonts.googleapis.com/css?family=Raleway:600&display=swap');
 
+.navbar {
+  font-family: 'Raleway', sans-serif;
+  background-color: #1a1e23;
+  display: flex;
+  flex-direction: row;
+  font-weight: 600;
+
+  @media (max-width: 500px) {
+    display: flex;
+    flex-direction: column;
+  }
+
+  nav {
+    z-index: 103 !important;
+  }
+
+  .logo {
+    font-size: 30px;
+    display: flex;
+    justify-content: flex-end;
+    // padding-left: 8.4rem;
+    padding-top: 0.2rem;
+    width: 15.1%;
+
+    @media (max-width: 600px) {
+      display: flex;
+      justify-content: center;
+      padding: 0;
+    }
+
+    @media (max-width: 500px) {
+      display: flex;
+      justify-content: center;
+      padding: 0;
+    }
+
+    a {
+      color: rgba(255, 255, 255, 1);
+      text-decoration: none;
+
+      span {
+        color: #e84c3d !important;
+      }
+    }
+  }
+
+  .nav {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+    align-items: center;
+    // border: 1px solid red;
```

Comment:
Life saver! The best advice I got in Lambda! once you're done with it, don't forget to delete it